charts/grafana/values.yaml

```diff
@@ -9,7 +9,7 @@ tolerations: []
 images:
   grafana:
     repository: quay.io/astronomer/ap-grafana
-    tag: 8.4.5-1
+    tag: 8.3.7-2
```

Comment:
This is wrong. We want master to be on the newer version. https://github.com/astronomer/issues/issues/4452

Reply:
The master has version [8.3.7-2](https://github.com/astronomer/ap-vendor/blob/main/grafana/version.txt)

Reply:
Fixed and updated to `8.4.5-2` as discussed.